File: src/main/java/com/zqh/fun/pp/getp.php
<?php
require 'phpQuery.php';
$query = phpQuery::newDocumentFile('http://me2-sex.lofter.com/tag/美女摄影?page=1');
$companies = pq('.pic')->find('img');
foreach ($companies as $img) {
    $url = pq($img)->attr('src');
    $file = fopen($url,"r");
    $w = fopen(time().mt_rand().'.jpg',"w");
    while (!feof($file)) {
        $f = fread($file, 1024);
        fwrite($w, $f);
    }
}

File: src/main/java/com/zqh/fun/pp/getp.rb
#encoding: utf-8
require 'rubygems'
require 'mechanize'
require 'open-uri'
Mechanize.new.get('http://me2-sex.lofter.com/tag/美女摄影?page=1').images.each do |img|
  File.open("./images/#{File.basename(img.url.to_s)}", 'w') { |f| f << open(img.url.to_s).read }
end
